### PR TITLE
Update community-handbook.md - adding Alex as RPM

### PR DESCRIPTION
**Pull request by aleesteele**
@AlexandraAAJ - adding you here. I was looking for references for a chapter while writing today, and found this omission!

### CODE REVIEW

File: book/website/community-handbook/community-handbook.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Here are a few suggestions for a good first contributions:
 
 The co-lead investigators for this project are [Dr Kirstie Whitaker](https://whitakerlab.github.io/about), who leads the Tools, Practices and Systems Research Programme at [The Alan Turing Institute](http://turing.ac.uk) and [Dr. Malvika Sharan](https://about.me/malvikasharan), Senior Researcher at the [Alan Turing Institute](http://turing.ac.uk).
 
-[Anne Lee Steele](https://www.turing.ac.uk/people/researchers/anne-lee-steele) is the Community Manager.
+[Anne Lee Steele](https://www.turing.ac.uk/people/researchers/anne-lee-steele) is the Research Community Manager. [Alexandra Araujo Alvarez](https://www.turing.ac.uk/people/programme-management/alexandra-araujo-alvarez) is the Research Project Manager.
 
 You can contact them on the community Gitter Channel [https://gitter.im/the-turing-way/the-turing-way](https://gitter.im/the-turing-way/the-turing-way).
```
Comment (AlexandraAAJ) [inline book/website/community-handbook/community-handbook.md:61]: Should this be removed?

**Review by AlexandraAAJ — CHANGES_REQUESTED** [review]

File: book/website/community-handbook/community-handbook.md (hunk 1 above)
Comment (aleesteele) [reply book/website/community-handbook/community-handbook.md:61]: Which bit @AlexandraAAJ? The Gitter channel?

Comment (AlexandraAAJ) [reply book/website/community-handbook/community-handbook.md:61]: Yes, if you're not using it anymore. Or mention that we are no longer using it.

Comment (aleesteele) [inline book/website/community-handbook/community-handbook.md:61]:
```suggestion
You can contact them on the community Slack workspace or directly by email.
```

Comment (aleesteele) [reply book/website/community-handbook/community-handbook.md]: Done! Changed to slack @AlexandraAAJ :)

**Pull request merged**